core/src/main/java/org/owasp/dependencycheck/analyzer/GolangModAnalyzer.java

```diff
@@ -286,7 +286,7 @@ protected void analyzeDependency(Dependency dependency, Engine engine) throws An
                 ProcessReader processReader = new ProcessReader(process, processor)) {
             processReader.readAll();
             final String error = processReader.getError();
-            if (error != null) {
+            if (error != null && !error.trim().isEmpty()) {
```

Comment:
Wouldn't a better solution just be:

```suggestion
            if (!StringUtils.isBalnk(error)) {
```

Reply:
Sure. The same convention was used in another class method and I missed that.